snapshots/run-android.sh

```diff
@@ -1,12 +1,17 @@
-# appium &
-# APPIUM_PID=$!
-emulator @Nexus_5X_API_25 -no-boot-anim &
+appium &
+APPIUM_PID=$!
+
+/Users/kourosaliabadi/Library/Android/sdk/tools/emulator @Nexus_5X_API_22 -no-boot-anim &
```

Comment:
Should this path be bound to Kouros's machine?

Reply:
lol probably not